### DISCUSSION TITLE
why I am not getting a refund?

### DISCUSSION THREAD

**Discussion by mtomczak123** [twbs/bootstrap#37352] [General]
My Order #117997. I wrote 3 emails (requests: 15206, 15133, 15138) No reaction. Give me my money back

**Comment by julien-deramond** [comment]
I've closed https://github.com/twbs/bootstrap/issues/37353 since it is not an issue linked to the source code of Bootstrap, and so this repository.
Regarding this order. What is the topic here? Is it related to [the Themes](https://themes.getbootstrap.com/), [the Swag Store](https://cottonbureau.com/people/bootstrap), something else?

**Comment by mtomczak123** [comment]
You don't represent Bootstrap?

I bought a theme but I want to quit and get my money back

 

I already sent 3 e-mails on this. No response

 

 

 

Hi there. Your recent order on Bootstrap Themes has been completed.

Downloads

ProductExpiresDownload
Blogzine – Blog and Magazine Bootstrap 5 ThemeNeverTheme File

Order #117997

ProductQuantityPrice
Blogzine – Blog and Magazine Bootstrap 5 Theme
Vendor: Webestica
License Type:
Multisite License
Theme File
Support
1$149.00
Subtotal:$149.00
Payment method:Credit Card
Total:$149.00

 

 

Od: Julien Déramond ***@***.***>
Odpowiedź do: twbs/bootstrap ***@***.***>
Data: niedziela, 23 października 2022 o 14:46
Do: twbs/bootstrap ***@***.***>
DW: Mariusz Tomczak ***@***.***>, Author ***@***.***>
Temat: Re: [twbs/bootstrap] why I am not getting a refund? (Discussion #37352)

 

I've closed #37353 since it is not an issue linked to the source code of Bootstrap, and so this repository.
Regarding this order. What is the topic here? Is it related to the Themes, the Swag Store, something else?

**Comment by mdo** [comment]
The Themes are separate from this open source project, we don't maintain or manage that business here. I'm sorry for the trouble with your refund though. Can you forward me your email to the Themes team, markdotto@gmail.com?

**Reply by mdo** [reply]
@mtomczak123 I've reached out to the team despite not receiving an email from you and the Themes team has confirmed it's been refunded. So sorry this took so long to get figured out!

**Comment by mtomczak123** [comment]
I forwarded you all the emails i sent to themes

https://i.ibb.co/4jtbx1H/Zrzut-ekranu-2022-10-26-o-06-48-41.jpg

 

I didn't get any refund

 

 

Od: Mark Otto ***@***.***>
Odpowiedź do: twbs/bootstrap ***@***.***>
Data: środa, 26 października 2022 o 01:09
Do: twbs/bootstrap ***@***.***>
DW: Mariusz Tomczak ***@***.***>, Mention ***@***.***>
Temat: Re: [twbs/bootstrap] why I am not getting a refund? (Discussion #37352)

 

@mtomczak123 I've reached out to the team despite not receiving an email from you and the Themes team has confirmed it's been refunded. So sorry this took so long to get figured out!
